lib/atlas/initializer_input.rb

```diff
@@ -0,0 +1,23 @@
+module Atlas
+  class InitializerInput
+    include ActiveDocument
+
+    DIRECTORY = 'initializer_inputs'
+
+    attribute :query,           String
+    attribute :share_group,     Symbol
+    attribute :priority,        Integer, default: 0
+    attribute :unit,            String
+    attribute :user_value,      Float
```

Comment:
That is not an attribute of the `InitialiserInput`. It is instead part of a derived dataset's usage of an init. input.